README.md

```diff
@@ -1,2 +1,35 @@
 # helper-bot
+05/19/2021
+
+
 Use Discord Slash Commands to offer people helpful advice
+
+To add your bot to your discord server with the appropriate permissions, ensure that the permission for applications.commands exists in addition to the bot OAUTH scope. Without this you will receive 
+
+    "403 Forbidden (error code: 50001): Missing Access"
+
+
+Dependencies: 
```

Comment:
```suggestion
### Dependencies: 
```

Maybe make it a Heading? Looks much cleaner and easily searchable with GitHub's new thing